<launch>
    <node ns="front_left" pkg="motor_control" type="talon_node" name="talon_srx" respawn="false">
        <param name="id" value="1" />
    </node>
    <node ns="front_right" pkg="motor_control" type="talon_node" name="talon_srx" respawn="false">
        <param name="id" value="2" />
    </node>
    <node ns="back_left" pkg="motor_control" type="talon_node" name="talon_srx" respawn="false">
        <param name="id" value="3" />
    </node>
    <node ns="back_right" pkg="motor_control" type="talon_node" name="talon_srx" respawn="false">
        <param name="id" value="4" />
    </node>
</launch>